juju/juju-gui-charm
# pip install the juju-gui tarball. Still non-functional.

QA: 

```
juju bootstrap
make deploy
juju status       # Ensure no errors
juju ssh juju-gui/0
pip list             # Ensure juju-gui (1.8.1000)
```
